### ISSUE TITLE
`att` method of `BaseSynth` is not recognised on Synth object

### ISSUE THREAD

**Issue by SUonGH** [sdfordham/pysyncon#38]
When trying to replicate the 'germany' example, I get the following error:
```
Traceback (most recent call last):
  File "/Users/<...>/test/german_reunification_example.py", line 89, in <module>
    print(synth.att(time_period=range(1990, 2004)))
AttributeError: 'Synth' object has no attribute 'att'
```

**Comment by SUonGH** [comment]
The example I ran was exactly the one you provided, but as a python script.
For completion ...
```
import pandas as pd
from pysyncon import Dataprep, Synth

df = pd.read_csv("german_reunification.csv")

dataprep_train = Dataprep(
    foo=df,
    predictors=["gdp", "trade", "infrate"],
    predictors_op="mean",
    time_predictors_prior=range(1971, 1981),
    special_predictors=[
        ("industry", range(1971, 1981), "mean"),
        ("schooling", [1970, 1975], "mean"),
        ("invest70", [1980], "mean"),
    ],
    dependent="gdp",
    unit_variable="country",
    time_variable="year",
    treatment_identifier="West Germany",
    controls_identifier=[
        "USA",
        "UK",
        "Austria",
        "Belgium",
        "Denmark",
        "France",
        "Italy",
        "Netherlands",
        "Norway",
        "Switzerland",
        "Japan",
        "Greece",
        "Portugal",
        "Spain",
        "Australia",
        "New Zealand",
    ],
    time_optimize_ssr=range(1981, 1991),
)


synth_train = Synth()
synth_train.fit(dataprep=dataprep_train)

dataprep = Dataprep(
    foo=df,
    predictors=["gdp", "trade", "infrate"],
    predictors_op="mean",
    time_predictors_prior=range(1981, 1991),
    special_predictors=[
        ("industry", range(1981, 1991), "mean"),
        ("schooling", [1980, 1985], "mean"),
        ("invest80", [1980], "mean"),
    ],
    dependent="gdp",
    unit_variable="country",
    time_variable="year",
    treatment_identifier="West Germany",
    controls_identifier=[
        "USA",
        "UK",
        "Austria",
        "Belgium",
        "Denmark",
        "France",
        "Italy",
        "Netherlands",
        "Norway",
        "Switzerland",
        "Japan",
        "Greece",
        "Portugal",
        "Spain",
        "Australia",
        "New Zealand",
    ],
    time_optimize_ssr=range(1960, 1990),
)


synth = Synth()
synth.fit(dataprep=dataprep, custom_V=synth_train.V)

print(synth.weights())

synth.path_plot(time_period=range(1960, 2004), treatment_time=1990)
synth.gaps_plot(time_period=range(1960, 2004), treatment_time=1990)
print(synth.att(time_period=range(1990, 2004)))
print(synth.summary())
```

**Comment by sdfordham** [comment]
This is published now to PyPI as [@v1.2.0](https://github.com/sdfordham/pysyncon/releases/tag/v1.2.0)

Run `python -m pip install pysyncon==1.2.0` to get it.

**Comment by SUonGH** [comment]
Thanks, this worked.
